worker.go

```diff
@@ -103,8 +106,10 @@ func rx(
 	// Create the event manager.
 	eventManager := NewEventManager(id, returnURL, responseInterval, w)
 
+	schema := getPlaybookDispatcherSchema()
+
 	// Create the playbook runner.
-	runner := ansible.NewRunner(correlationID, 60*time.Second)
+	runner := ansible.NewRunner(correlationID, 60*time.Second, schema)
```

Comment:
Is it necessary to pass `schema` as a argument to `NewRunner`? Is it related somehow with the TODO on the line 359?

Reply:
Yes -- ideally this schema will be downloaded dynamically in case it ever changes.